$NetBSD: patch-rebar.config,v 1.2 2016/01/16 18:46:30 fhajny Exp $

Deps are handled by pkgsrc.
Look up p1_stun includes properly.

--- rebar.config.orig	2016-01-05 14:09:44.000000000 +0000
+++ rebar.config
@@ -1,11 +1,7 @@
 {erl_opts, [{i, "include"},
-	    {i, "deps/p1_stun/include"},
+	    {i, "@LOCALBASE@/lib/erlang/lib/p1_stun/include"},
 	    {i, "../p1_stun/include"}]}.
 
-{deps, [{p1_stun, ".*", {git, "https://github.com/processone/stun", {tag, "0.9.0"}}},
-	{p1_tls, ".*", {git, "https://github.com/processone/tls", {tag, "1.0.0"}}},
-	{p1_utils, ".*", {git, "https://github.com/processone/p1_utils", {tag, "1.0.2"}}}]}.
-
 %% Local Variables:
 %% mode: erlang
 %% End:
